InterfaceParser/model/double.py

```diff
@@ -0,0 +1,18 @@
+"""Floating-point type.
+
+"""
+
+
+class Double:
```

Comment:
Was there an issue with naming this class "Float"?

Reply:
this class came from legacy sdl_core parser/generator. Yes we can rename it, but as this parser/model going to be used for all generators (including sdl_core) the appropriate refactoring will be required for all generators. Consider whether this renaming required?

Reply:
Yes sorry let me give this one some thought. I dont want to introduce breaking changes unnecessarily.

Reply:
Ok after some discussion with team members, I am still going to ask for naming "Double" to be changed to "Float". The generator in core that will be compatible with this submodule should be adjusted to use "Float" instead of "Double".

Regardless of changing this naming, the existing parsers and generators in Core should be noted as deprecated and unmodified.

The new/refactored Core generator that uses this submodule parser should be a new set of files and be considered a new feature to avoid a breaking change.

Reply:
It is possible to rename the class from `Double` to `Float` and as @o-mishch said this will lead to renaming the occurrences in other generators as well. 
@TinaKleczka @kshala-ford we need your assistance here. It seems we are disconnected what should be done regarding the `sdl_core` Generator part.
As you could see here https://github.com/smartdevicelink/sdl_core/pull/3213 we have done with this by splitting the `InterfaceGenerator` to Parser/Generator, moving the parser part here and rejoin this into old `sdl_core`'s `InterfaceGenerator` as Git submodule, leaving the code as-is, only do changes for Python 3 compatibility. We didn't do any refactoring in the old `sdl_core` `InterfaceGenerator`, because it seems we cover the proposal's requirements, the `InterfaceGenerator` includes `rpc_spec` scripts and is compatible with Python 3. So we don't pretty sure what @JackLivio means here under the `deprecated` notices and a new set of files of the new/refactored Core generator? Does it mean the `sdl_core` generator should be fully refactored?

Reply:
I think minimum the core generator should be refactored to work with the changes in the RPC spec parsers. Maybe in Core and this PR the usage of "Double" should be marked deprecated in favor of using the "Float" parser type instead. Both Float and Double classes could be included in this PR to accommodate preventing a breaking change.